docs/getting-started.md

```diff
@@ -1,105 +1,177 @@
+abort a rollout at any time during an update:
 
-Argo Rollouts has a [kubectl plugin](features/kubectl-plugin.md) to help automate operations like promoting a rollout through a step. The installation instructions are [here](features/kubectl-plugin.md#installation).
+```shell
+kubectl argo rollouts abort rollouts-demo
+```
+
+When a rollout is aborted, it will scale up the "stable" version of the ReplicaSet (in this
+case the yellow image), and scale down any other versions. Although the stable version of the
+ReplicaSet may be running and is healthy, the overall rollout is still considered `Degraded`, 
+since the desired version (the red image) is not the version which is actually running.
 
-Once the plugin is installed, the user can run the following command to promote the rollout through the pause step:
+![Aborted Rollout](getting-started/basic/aborted-rollout.png)
 
-```bash
-kubectl argo rollouts promote example-rollout
+In order to make Rollout considered Healthy again and not Degraded, it is necessary to change the
+desired state back to the previous, stable version. This typically involves running `kubectl apply`
+against the previous Rollout spec. In our case, we can simply re-run the `set image` command using
+the previous, "yellow" image.
 
+```yaml
+kubectl argo rollouts set image rollouts-demo \
+  rollouts-demo=argoproj/rollouts-demo:yellow
 ```
 
-At this point, the Rollout has executed all the steps to transition to a new version. As a result, the new ReplicaSet is considered the new stable ReplicaSet, and the previous ReplicaSet will be scaled down. The Rollout will repeat these steps when the Pod Spec Template is changed again.
+After running this command, you should notice that the Rollout immediately becomes Healthy, and
+there is no activity with regards to new ReplicaSets becoming created.
+
+![Healthy Rollout (Revision 4)](getting-started/basic/healthy-rollout-rev4.png)
+
+When a Rollout has not yet reached its desired state (e.g. it was aborted, or in the middle of
+an update), and the stable manifest were re-applied, the Rollout detects this as a rollback 
+and *not* a update, and will fast-track the deployment of the stable ReplicaSet by skipping
+analysis, and the steps.
+
+## Summary
+
+In this guide, we have learned basic capabilities of Argo Rolouts, including:
+
+* Deploying a rollout
+* Performing a canary update
+* Manual promotion
+* Manual abortion 
+
+The Rollout in this basic example did not utilize a ingress controller or service mesh provider
+to route traffic. Instead, it used normal Kubernetes Service networking (i.e. kube-proxy) to achieve
+an *approximate* canary weight, based on the closest ratio of new to old replica counts.
+As a result, this Rollout had a limitation in that it could only achieve a minimum canary
+weight of 20%, by scaling 1 of 5 pods to run the new version. In order to achieve much
+finer grained canarys, a ingress controller or service mesh is necessary.
```

Comment:
Should we add a note about what happens with a user has a weight that doesn't divide evenly?

Reply:
I think I want to describe this when revamping the canary strategy and explaining the difference when trafficRouting is omitted.